pipelines/DRP.yaml

```diff
@@ -21,9 +21,31 @@ tasks:
       doPhotometry: false
       applyColorTerms: false
       python: |
-        # This override is temporary until DM-29008.
-        from lsst.utils import getPackageDir
-        config.load(os.path.join(getPackageDir("jointcal"), "tests", "config", "config-gen3.py"))
```

Comment:
I think it might be worth adding a comment on where these variables came from.

Reply:
We should have a TO-DO on DM-29008 that once that is done then these overrides are no longer necessary.

Reply:
You mean just saying "These defaults were copied from the config-gen3.yaml file provided in the tests/config directory of the jointcal package"?

Reply:
Yup, and the TODO.